transport-classes-kqueue/src/main/java/io/netty/channel/kqueue/AbstractKQueueServerChannel.java

```diff
@@ -32,6 +32,11 @@ public abstract class AbstractKQueueServerChannel extends AbstractKQueueChannel
 
     private final EventLoopGroup childEventLoopGroup;
 
+    // Will hold the remote address after accept(...) was successful.
+    // We need 24 bytes for the address as maximum + 1 byte for storing the capacity.
+    // So use 26 bytes as it's a power of two.
```

Comment:
This old comment again.

Reply:
you are right... This code was just moved and is not new. Let me just fix the comment 🤦

Reply:
Actually let me just allocate the exact size https://github.com/netty/netty/pull/15973